xapian-letor/api/featurelist.cc

```diff
@@ -31,8 +32,7 @@ namespace Xapian {
 
 FeatureList::FeatureList() {
 
-    std::cout << "Initialising DEFAULT list of features" << endl;
-
+    LOGCALL_CTOR(API, "FeatureList", NO_ARGS);
     feature.push_back(new TfFeature());
```

Comment:
Blank lines at the start of blocks are better avoided too.  Again, probably just avoid introducing more, and we can do a mass clean-up commit at some point.